packages/mediacenter/kodi/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="kodi"
-PKG_VERSION="19.4-Matrix"
-PKG_SHA256="cc026f59fd6e37ae90f3449df50810f1cefa37da9444e1188302d910518710da"
+PKG_VERSION="181070c647425a8730fb7d8f1c349cd750e40021"
```

Comment:
Is there a reason to update to a non release in a release branch?

Reply:
Same reason as with the previous bumps to pre-release versions https://github.com/LibreELEC/LibreELEC.tv/commits/libreelec-10.0/packages/mediacenter/kodi/package.mk - getting bugfixes and increasing test-coverage while kodi release is being delayed